android/src/main/java/com/tailscale/ipn/ui/localapi/LocalAPIClient.kt

```diff
@@ -23,6 +25,10 @@ class LocalApiClient(private val scope: CoroutineScope) {
     }
 
     companion object {
+        // TODO(oxtoacart): I'm using Dispatchers.IO here because using the lifecycle scope seems to
+        // block the UI thread. Can we marry lifecycle scope with Dispatchers.IO somehow?
+        private val ioScope = CoroutineScope(Dispatchers.IO + Job())
```

Comment:
My switch to lifecycle scope really slowed things down because our calls to Go where effectively executing on the UI thread. This walks that back.

Reply:
I found a better solution than the above.